src/main/java/frc/robot/drive/DriveSubsystem.java

```diff
@@ -49,29 +44,29 @@ public DriveSubsystem(PoseScheduler poseScheduler) {
       swerve = new SwerveParser(new File(Filesystem.getDeployDirectory(), "swerve")).createSwerveDrive(
         Constants.Drive.MAXIMUM_VELOCITY, 
         SwerveMath.calculateDegreesPerSteeringRotation(Constants.Drive.STEER_GEAR_RATIO, Constants.Drive.PULSE_PER_ROTATION), 
-        SwerveMath.calculateMetersPerRotation( Constants.Drive.WHEEL_DIAMETER_METERS, Constants.Drive.DRIVE_GEAR_RATIO, Constants.Drive.PULSE_PER_ROTATION)
+        SwerveMath.calculateMetersPerRotation(Constants.Drive.WHEEL_DIAMETER_METERS, Constants.Drive.DRIVE_GEAR_RATIO, Constants.Drive.PULSE_PER_ROTATION)
       );
     } catch (IOException e) {
-      System.out.println("Swerve drive configuration file could not be found at " + Filesystem.getDeployDirectory() + "/swerve");
       e.printStackTrace();
+      throw new Error("Swerve drive configuration could not be loaded from " + Filesystem.getDeployDirectory() + "/swerve", e);
     }
 
-    swerve.chassisVelocityCorrection = false;    
+    swerve.chassisVelocityCorrection = false; //TODO test out setting this
     swerve.setHeadingCorrection(true);
     initDashboard();
   }
 
   private void initDashboard() {
-    dashboard = Shuffleboard.getTab("Drive");
+    ShuffleboardTab dashboard = Shuffleboard.getTab("Drive");
     dashboard.addString("Current Command", this::getCommandName);
 
     dashboard.add("Rotation PID", rotationController);
     dashboard.addDouble("Yaw", () ->swerve.getYaw().getDegrees());
-    dashboard.addDouble("Target angle", () -> targetAngle == null ? -1 : targetAngle);
+    dashboard.addDouble("Target angle", () -> rotationTarget == null ? -1 : rotationTarget.getDegrees());
```

Comment:
Different problem now. We should  not rely on -1 being no target. I think if there is no target we should just log this as the current angle, we can compare with the Boolean as well to know when something is actually a target.